### PR TITLE
Workplan labels

### PR DESCRIPTION
**Pull request by ilcsik**

### CODE REVIEW

File: EquiTrack/EquiTrack/factories.py [hunk 1]
```diff
@@ -191,13 +193,76 @@ class Meta:
...
+
+    name = factory.Sequence(lambda n: 'Label {}'.format(n))
+
+
+class ResultWorkplanPropertyFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = workplan_models.ResultWorkplanProperty
+
+    workplan = factory.SubFactory(WorkplanFactory)
+    result_type = factory.SubFactory(ResultTypeFactory)
```
Comment (robertavram) [inline EquiTrack/EquiTrack/factories.py]: I believe result type needs to be on "Result"

File: EquiTrack/workplan/models.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+from users.models import Section
+from locations.models import Location
+from partners.models import PartnerOrganization
+from reports.models import ResultStructure, ResultType
+
+
+class Comment(models.Model):
+    author = models.ForeignKey(User, related_name='+')
+    tagged_users = models.ManyToManyField(User, blank=True, related_name='+')
```
Comment (robertavram) [inline EquiTrack/workplan/models.py]: good thought on the 'related_name='+'"
👍

File: EquiTrack/workplan/models.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    timestamp = models.DateTimeField(auto_now_add=True)
+    text = models.TextField()
+
+    # This model will have relations to multiple other models. One FK would not be enough
+    content_type = models.ForeignKey(ContentType, null=True, blank=True, on_delete=models.CASCADE)
```
Comment (robertavram) [inline EquiTrack/workplan/models.py]: are we keeping contenttype?

File: EquiTrack/EquiTrack/factories.py (hunk 1 above)
Comment (ilcsik) [inline EquiTrack/EquiTrack/factories.py]: For some reason I had this link on my diagram sketch that I did when we discussed, but I can't really recall, could be a mistake. Shall I remove?

Comment (robertavram) [inline EquiTrack/EquiTrack/factories.py]: Yes. I think you have it right on the model.. it's just in the factory

Comment (ilcsik) [inline EquiTrack/EquiTrack/factories.py]: Removed in https://github.com/unicef/etools/pull/137/commits/0d3f4ea668be50843421ad3e47abc662907a917d

File: EquiTrack/workplan/models.py (hunk 3 above)
Comment (ilcsik) [inline EquiTrack/workplan/models.py]: @tejfel ?

Comment (tejfel) [inline EquiTrack/workplan/models.py]: No. The base branch was updated already. This branch just have to be rebased on the top of that.

**Comment by ilcsik** [comment]
Merged to https://github.com/unicef/etools/tree/workplan-main

**Pull request closed without merging**